src/components/Dashboard/SortList/SortList.scss

```diff
@@ -0,0 +1,43 @@
+.btn{
+    &.btn-no-outline{
+        &:focus,
+        &:active {
+            outline: none !important;
+            box-shadow: none !important;
+        }
+    }
+
+    &-custom-light{
+        &:active,
+        &.active {
+            background-color: #d8d9da !important;
+            border-color: #d8d9da !important;
+        }
+    }
+}
+
+.dropdown-toggle{
+    &.no-arrow{
+        &::after{
+            display: none !important;
+        }
+    }
+}
+.dropdown-item{
+    &.dont-highlight{
+        &:active {
+            background-color: #f8f9fa;
+        }
+    }
+}
+
+img.icon{
+    height: 1.5em;
+}
+
+
+.dropdown-pull-up{
```

Comment:
It is customary to put spaces after selectors in CSS. This code is a little ugly.